services/vmalert/alerting_rules.go

```diff
@@ -53,19 +59,52 @@ func (s *AlertingRules) ValidateRules(ctx context.Context, rules string) error {
 
 // ReadRules reads current rules from FS.
 func (s *AlertingRules) ReadRules() (string, error) {
-	b, err := ioutil.ReadFile(alertingRulesFile)
-	if err != nil && !os.IsNotExist(err) {
-		return "", err
-	}
-	return string(b), nil
+	// TODO: this method supposed to return external alertmanager rules.
```

Comment:
Not sure why you removed the implementation

Reply:
Reverted